### PR TITLE
[MRG] fix for annotations when plotting

### PR DESCRIPTION
**Pull request by jasmainak**
[This](https://mail.nmr.mgh.harvard.edu/pipermail//mne_analysis/2016-August/003534.html) came up in the mailing list. Here's a fix so that this problem doesn't happen again

cc @jaeilepp

### CODE REVIEW

File: mne/viz/tests/test_raw.py [hunk 1]
```diff
@@ -121,6 +122,12 @@ def test_plot_raw():
                             kind='release')
 
             plt.close('all')
+        # test if meas_date has only one element
+        raw.info['meas_date'] = np.array([raw.info['meas_date'][0]],
+                                         dtype=np.int32)
+        raw.annotations = Annotations([1 + raw.first_samp / raw.info['sfreq']],
+                                      [5], ['bad'])
+        raw.plot()
```
Comment (Eric89GXL) [inline mne/viz/tests/test_raw.py]: you should probably have a `plt.close('all')` after this to free memory

**Comment by Eric89GXL** [comment]
LGTM but we should let @jaeilepp have a look at this before merge

**Comment by jaeilepp** [comment]
+1 for merge after addressing comment by @Eric89GXL

**Comment by jasmainak** [comment]
> +1 for merge after addressing comment by @Eric89GXL

done :)

**Pull request merged**